test-commons/src/main/java/gov/cms/qpp/test/helper/AwsTestHelper.java

```diff
@@ -0,0 +1,62 @@
+package gov.cms.qpp.test.helper;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.dynamodbv2.model.ScanRequest;
+import com.amazonaws.services.dynamodbv2.model.ScanResult;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.ObjectListing;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import com.google.common.collect.Lists;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A helper class for aws testing
+ */
+public class AwsTestHelper {
+	private static AmazonDynamoDB dynamoClient = AmazonDynamoDBClientBuilder.standard().build();
+	private static AmazonS3 s3Client = AmazonS3ClientBuilder.standard().withRegion(Regions.US_EAST_1).build();
+	public static final String TEST_DYNAMO_TABLE_NAME = "qpp-qrda3converter-acceptance-test";
+	private static final String TEST_S3_BUCKET_NAME = "qpp-qrda3converter-acceptance-test";
+
+	/**
+	 * Cleans dynamodb items until via batch scan and delete for performance purposes
+	 */
+	public static void cleanDynamoDb() {
+		ScanResult scanResult = dynamoClient.scan(TEST_DYNAMO_TABLE_NAME, Lists.newArrayList("Uuid"));
+		List<Map<String, AttributeValue>> metadataList = scanResult.getItems();
+		while (scanResult.getLastEvaluatedKey() != null && !scanResult.getLastEvaluatedKey().isEmpty()) {
+			scanResult = dynamoClient.scan(new ScanRequest().withTableName(TEST_DYNAMO_TABLE_NAME).withAttributesToGet("Uuid")
+				.withExclusiveStartKey(scanResult.getLastEvaluatedKey()));
+			metadataList.addAll(scanResult.getItems());
+		}
+
+		metadataList.forEach(map -> dynamoClient.deleteItem(TEST_DYNAMO_TABLE_NAME, map));
+	}
+
+	/**
+	 * Cleans up the test s3 bucket by batch for performance purposes
+	 */
+	public static void cleanS3() {
```

Comment:
do we need to implement any delay between batches for aws rate limiting?

Reply:
I don't believe so because this is strictly an acceptance test dynamodb table and s3 bucket. When I originally ran this we had quite a number of files but now they will always be clean after the tests run. We can implement a delay if needed but I don't think we have high enough load to really need it.